source/administration/mmctl-cli-tool.rst

```diff
@@ -101,7 +101,7 @@ To use local mode, the Mattermost server first needs to `have local mode enabled
 Using local mode
 ----------------
 
-You need to append ``--local`` to the command you want to use or set the environment variable as ``MMCTL_LOCAL=true``.
+You need to append ``--local`` to the command you want to use or set the environment variable as ``MMCTL_LOCAL=true``. If you want to use another socket file instead of the default one, you need to set it to ``MMCTL_LOCAL_SOCKET_PATH`` environment variable. Note that this file should be same with the `server configuration <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`.
```

Comment:
```suggestion
You need to append ``--local`` to the command you want to use, or set the environment variable as ``MMCTL_LOCAL=true``. To use a socket file other than the default, you need to set the environment variable to ``MMCTL_LOCAL_SOCKET_PATH``. This file should match the `server configuration setting <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`.
```

Reply:
@isacikgoz When using a socket file other than the default, we're currently saying that the file _should_ match the `local mode socket location` server configuration setting. Are we making a recommendation or is this a requirement? If it's the latter, I'd like to strengthen the language from "should" to "must".

Reply:
Yep this is a requirement, we can strengthen the wordage there 👍

Reply:
Thanks! Suggestion updated!

Reply:
Awesome! thanks @cwarnermm 🎉